mirar/pipelines/winter/validator.py

```diff
@@ -53,24 +60,50 @@ def poor_astrometric_quality_rejector(batch: ImageBatch) -> ImageBatch:
     above 1.0 arcsec
     3. Rejects images with median FWHM above 6.0 arcsec
     """
-    scamp_astrrms_threshold_arcsec = 0.5
-    astrometric_unc_threshold_arcsec = 1.0
+    astrometric_unc_threshold_arcsec = 3.0
     fwhm_threshold_arcsec = 6.0
     for image in batch:
-        if (image["ASTRRMS1"] > scamp_astrrms_threshold_arcsec / 3600) | (
-            image["ASTRRMS2"] > scamp_astrrms_threshold_arcsec / 3600
-        ):
-            raise PoorAstrometryError(
-                f"RMS astrometric error from Scamp "
-                f"({image['ASTRRMS1']}, {image['ASTRRMS2']}) is above threshold"
-            )
-
         if image["ASTUNC"] > astrometric_unc_threshold_arcsec / 3600:
             raise PoorAstrometryError(
                 f"Uncertainty in astrometric solution from Scamp "
-                f"({image['ASTUNC']}) is above threshold"
+                f"({image['ASTUNC']*3600}) arcsec is above threshold "
+                f"{astrometric_unc_threshold_arcsec} arcsec"
             )
 
         if image["FWHM_MED"] > fwhm_threshold_arcsec:
             raise PoorFWHMError(f"FWHM ({image['FWHM_MED']}) is above threshold")
     return batch
+
+
+def is_condensation_in_image(data: np.ndarray, header: fits.Header) -> bool:
```

Comment:
Why does this not just run on image objects?

Reply:
Ah I thought we may want to copy/use the function as is elsewhere, so kept it independent of mirar-specifics.

Reply:
Could you just copy this version of the function into the "drpclone" and then change it? It took a lot of effort to convert all functions to using mirar data types, but that is important because it gives us flexibility to change those data types without impacting functions like this. E.g one day we might replace astropy headers with python dictionaries. Same with the numpy arrays, maybe there's some else which is faster. If you just use the Image and get_header/get_data, the code will still work fine no matter what backend changes we make to Image objects.